src/main.js

```diff
@@ -11,6 +11,8 @@ let base = require('taskcluster-base');
 
 let workerType = require('./worker-type');
 let secret = require('./secret');
+let workerState = require('./worker-state');
```

Comment:
You should rebase your patch to the latest version of the upstream master branch.  There was a change that I landed today that deletes the WorkerState entity, and it looks like you're adding it back here.